### DISCUSSION TITLE
how to save raw byte[] with color mode to png image?

### DISCUSSION THREAD

**Discussion by srxqds** [SixLabors/ImageSharp#2111] [Show and tell]
I have pixel bytes with color mode, like grayscale, how can I save it to png.

**Comment by alan0428a** [comment]
You can reference the documentation [here](https://docs.sixlabors.com/articles/imagesharp/pixelbuffers.html#loading-raw-pixel-data-into-an-imaget)

**Comment by JimBobSquarePants** [comment]
Hey @srxqds please don't use the Ideas category for questions. It's for feature suggestions only.
